react/Labs/ExperimentalModal/index.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+
+import Dialog from '@material-ui/core/Dialog'
+import DialogTitle from '@material-ui/core/DialogTitle'
+import DialogActions from '@material-ui/core/DialogActions'
+
+import withMobileDialog from '@material-ui/core/withMobileDialog'
+
+import AppTitle from '../../AppTitle'
+
+export const ExperimentalModal = props => {
+  const { dismissAction, fullScreen, children, open } = props
+  return (
+    <Dialog
+      open={open}
+      onClose={dismissAction}
+      fullScreen={fullScreen}
+      aria-labelledby="dialog-title"
+      scroll={'paper'}
```

Comment:
Generally speaking we could probably spread whatever props we get here, couldn't we? Having default prop values is a great addition, especially the `open`, but I don't see the value in renaming `onClose` to `dismissAction` for example.

Reply:
What it's the goal? Should we stay as close as possible to our actual modal? Or do we embrace fully the API from MUI? If yes, I can remove Modal* reference to only get Dialog (ExperimentalDialog for instance) and use the same props as Modal

Reply:
The goal is to have a modal with an API and a look that better suits or needs. I don't see a reason to force ourselves staying close to the previous API, in fact it may be quite limiting.